spec/models/category_product_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe CategoryProduct, type: :model do
+  describe 'database' do
+    it 'has columns' do
+      expect(subject).to have_db_column(:product_id).of_type(:integer)
+      expect(subject).to have_db_column(:category_id).of_type(:integer)
+      expect(subject).to have_db_column(:created_at).of_type(:datetime).with_options(null: false)
+      expect(subject).to have_db_column(:updated_at).of_type(:datetime).with_options(null: false)
+    end
+
+    it 'has indices' do
+      expect(subject).to have_db_index(:product_id)
+      expect(subject).to have_db_index(:category_id)
+    end
+  end
+
+  describe 'validations' do
```

Comment:
These are not validations actually
```suggestion
  describe 'relations' do
```

Reply:
I don't see this change applied. Are you going to change it?